### ISSUE TITLE
Can I make charts with FL Studio?

### ISSUE THREAD

**Issue by zwl1619** [KadeDev/Funkin#2600]
There is a song.

Can I make charts that using in FNF with FL Studio?

**Comment by ishsharm0** [comment]
No, you have to use FNF/kade Engine's charter to chart songs. You can use songs made in FL Studio, but not chart them directly in that app.

**Comment by RagdollClash** [comment]
You can use [SiIva Note Importer For FNF](https://github.com/PrincessMtH/SNIFF) to chart in FL Studio

**Comment by zwl1619** [comment]
@ActualMandM Does "SiIva Note Importer For FNF" have a tutorial?

**Comment by Dobby233Liu** [comment]
> @\ActualMandM Does "SiIva Note Importer For FNF" have a tutorial?

https://github.com/PrincessMtH/SNIFF/blob/master/_extra%20release%20files/README.txt
